makefile

```diff
@@ -1,220 +1,19 @@
-##########################################################################################################################
-# File automatically-generated by tool: [projectgenerator] version: [3.15.2] date: [Sun Jun 26 14:03:25 EDT 2022]
-##########################################################################################################################
-
-# ------------------------------------------------
-# Generic Makefile (based on gcc)
-#
-# ChangeLog :
-#	2017-02-10 - Several enhancements + project update mode
-#   2015-07-22 - first version
-# ------------------------------------------------
-
-######################################
-# target
-######################################
-TARGET = main
+THIS_DIR := $(shell readlink -f .)
+BUILD_DIR = $(THIS_DIR)/build
 
-DEVICE_DIRNAME= STM32G473CBTx
+all:
+	cd $(DEV) && make BUILD_DIR=$(BUILD_DIR)
```

Comment:
is DEV defined somewhere?

Reply:
I indented it to be passed as a CLI argument.

Like:
`make DEV=whatever`